featured-video/featured-video.php

```
@@ -0,0 +1,118 @@
+add_action( 'init', 'wpcomsp_featured_video_register_post_meta' );
+
+/**
+ * Render the featured video in place of the post featured image.
+ *
+ * @param string   $block_content The block content.
+ * @param array    $block         The block attributes.
+ * @param WP_Block $wp_block      The WP_Block instance.
+ *
+ * @return string The updated block content with the featured video.
+ */
+function wpcomsp_featured_video_render_post_featured_image( $block_content, $block, $wp_block ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$block'. (Unused Code Rules)

(UnusedFormalParameter)